### PR TITLE
Update lumina-config_sv.ts

### PR DESCRIPTION
**Pull request by bittin**
Updated Swedish Translation 98%

### CODE REVIEW

**Review by beanpole135 — CHANGES_REQUESTED** [review]
All of these translations are in the wrong spot and will not be used they way they are now.
The `type="unfinished"` tag needs to be removed, and the translated text put between the tags like this:
`<translation>This is translated text</translation>`

File: src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts [hunk 1]
```diff
@@ -782,12 +782,12 @@
...
         <location filename="../pages/getPage.cpp" line="55"/>
         <source>Sound Themeing</source>
-        <translation type="unfinished"></translation>
+        <translation>Ljud teman</translation>
     </message>
     <message>
         <location filename="../pages/getPage.cpp" line="55"/>
-        <source>Change basic sound settings</source>
-        <translation type="unfinished"></translation>
+        <source>Change grundläggande ljudinställningar</source>
```
Comment (beanpole135) [inline src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts:789]:
```suggestion
        <source>Change basic sound settings</source>
```
Do not change the "source" text - that could result in the translation not getting applied properly.

File: src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts [hunk 2]
```diff
@@ -951,67 +951,67 @@
...
         <location filename="../../../core/libLumina/LuminaXDG.cpp" line="616"/>
         <source>Settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Inställningar</translation>
     </message>
     <message>
         <location filename="../../../core/libLumina/LuminaXDG.cpp" line="617"/>
         <source>System</source>
-        <translation type="unfinished">System</translation>
+        <translation>System">System</translation>
```
Comment (beanpole135) [inline src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts:999]:
```suggestion
        <translation>System</translation>
```

File: src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts [hunk 3]
```diff
@@ -1147,12 +1147,12 @@
     <message>
         <location filename="../pages/page_compton.ui" line="32"/>
         <source>Disable Compositing Manager (session restart required)</source>
-        <translation type="unfinished"></translation>
+        <translation>unfinished</translation>
```
Comment (beanpole135) [inline src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts:1150]: This needs to be finished, or reverted back to having the `type="unfinished"` tag.

File: src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts [hunk 4]
```diff
@@ -1147,12 +1147,12 @@
...
     </message>
     <message>
         <location filename="../pages/page_compton.ui" line="39"/>
         <source>Only use compositing with GPU acceleration </source>
-        <translation type="unfinished"></translation>
+        <translation>unfinished</translation>
```
Comment (beanpole135) [inline src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts:1155]: This needs to be finished, or reverted back to having the `type="unfinished"` tag.

File: src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts [hunk 5]
```diff
@@ -1475,7 +1475,7 @@
     <message>
         <location filename="../pages/page_interface_desktop.ui" line="86"/>
         <source>Display Removable Media Icons</source>
-        <translation type="unfinished"></translation>
+        <translation>unfinished</translation>
```
Comment (beanpole135) [inline src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts:1478]: This needs to be finished, or reverted back to having the `type="unfinished"` tag.

File: src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts [hunk 6]
```diff
@@ -1585,7 +1585,7 @@
     <message>
         <location filename="../pages/page_main.cpp" line="64"/>
         <source>Desktop Defaults</source>
-        <translation type="unfinished"></translation>
+        <translation>unfinished</translation>
```
Comment (beanpole135) [inline src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts:1588]: This needs to be finished, or reverted back to having the `type="unfinished"` tag.

File: src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts [hunk 7]
```diff
@@ -1733,7 +1733,7 @@
     <message>
         <location filename="../pages/page_session_options.ui" line="220"/>
         <source>Window Manager</source>
-        <translation type="unfinished">Fönsterhanterare</translation>
+        <translation>Fönsterhanterare">Fönsterhanterare</translation>
```
Comment (beanpole135) [inline src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts:1736]:
```suggestion
        <translation>Fönsterhanterare</translation>
```

File: src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts [hunk 8]
```diff
@@ -1921,56 +1921,56 @@
     <message>
         <location filename="../pages/page_soundtheme.ui" line="14"/>
         <source>Form</source>
-        <translation type="unfinished">Formulär</translation>
+        <translation>Formulär">Formulär</translation>
```
Comment (beanpole135) [inline src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts:1924]:
```suggestion
        <translation>Formulär</translation>
```

File: src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts [hunk 9]
```diff
@@ -1921,56 +1921,56 @@
...
-        <translation type="unfinished"></translation>
+        <translation>Aktiverad</translation>
     </message>
     <message>
         <location filename="../pages/page_soundtheme.ui" line="36"/>
         <location filename="../pages/page_soundtheme.ui" line="81"/>
         <location filename="../pages/page_soundtheme.ui" line="126"/>
         <source>TextLabel</source>
-        <translation type="unfinished"></translation>
+        <translation>unfinished</translation>
```
Comment (beanpole135) [inline src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts:1938]: This needs to be finished, or reverted back to having the `type="unfinished"` tag.

**Review by beanpole135 — CHANGES_REQUESTED** [review]
Got a few issues with this one still.
I suggested changes for some of them, the others just need to be finished it looks like.

**Comment by q5sys** [comment]
This work has been redone with later PRs

**Pull request closed without merging**